var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import tempfile
+
+import llnl.util.tty as tty
+
+
+class PyJaxlib(PythonPackage, CudaPackage):
+    """XLA library for Jax"""
+
+    homepage = "https://github.com/google/jax"
+    url = "https://github.com/google/jax/archive/refs/tags/jaxlib-v0.1.74.tar.gz"
+
+    tmp_path = ""
+
+    version('0.1.74', sha256='bbc78c7a4927012dcb1b7cd135c7521f782d7dad516a2401b56d3190f81afe35')
+
+    depends_on('python@3.7:')
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
```

Comment:
Where do you see wheel listed? Actually, once #27798 is merged, all PythonPackages will have a dep on wheel

Reply:
I _think_ that's only used when running `python setup.py bdist_wheel` or `pip install`, but I could be wrong. Anyway, it doesn't really make a difference in the long run.